tf5to6server/doc.go

```diff
@@ -0,0 +1,12 @@
+// Translate a protocol version 5 provider server into protocol version 6.
+//
+// Supported protocol version 5 provider servers include any which implement
+// the github.com/hashicorp/terraform-plugin-go/tfprotov5.ProviderServer
+// interface, such as:
+//
+// - github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server
+// - github.com/hashicorp/terraform-plugin-mux/tf6to5server
```

Comment:
Should this be `github.com/hashicorp/terraform-plugin-mux/tf5to6server` ?

Reply:
`tf6to5server` implements the `tfprotov5.ProviderServer` interface 😉 

https://github.com/hashicorp/terraform-plugin-mux/blob/3b76c74adaef3b1256c9aabee746aa746a67b2d3/tf6to5server/tf6to5server.go#L38

Although now that I'm re-reading this, we probably shouldn't mention it because it'd be weird to use _both_ `tf6to5server` and `tf5to6server` since you can just serve the protocol version 6 server. What I meant to document was `tf5muxserver`. 😅 

```suggestion
// - github.com/hashicorp/terraform-plugin-mux/tf5muxserver
```